src/frontend/src/common/styles/support/style.ts

```diff
@@ -14,6 +14,7 @@ import {
 export const NarrowContainer = styled.div`
   max-width: 800px;
   margin: 90px auto;
+  padding-bottom: 50px;
```

Comment:
~Lord this is minor, but I would probably put this as just more margin than padding since I think that's closer to the original intent of the spacing block. I know we already have a `margin`, but we could increase the bottom margin by altering the shorthand use like so:~
```
  margin: 90px auto 140px;
```

Huh! That's one I didn't know about, I tried this out before actually putting in the comment -- nope, doesn't work! Looks like `margin-bottom` at the very end of the page (at least in Chrome) just gets obliterated. No need to have a margin if it's just a margin between an element and the void of nothing-ness? I dunno what the reasoning there is, but looks like `padding-bottom` works like you'd expect and actually does space it out.

That said, it does mean that the shorthand `90px` for top-bottom is only half-true here! It provides a real 90px up top, but the bottom 90px just gets eaten by the void, so we only have the 50px from padding-bottom. Go figure.

So definitely keep the padding-bottom, but maybe adjust its amount and/or adjust the `margin` shorthand so it's not implying something that actually doesn't happen? Like
```suggestion
  margin: 90px auto 0;
  padding-bottom: 50px;
```